libsplinter/src/admin/rest_api/actix/ws_register_type.rs

```diff
@@ -317,24 +231,10 @@ struct JsonAdminEvent {
     event_id: Option<i64>,
 }
 
-// Conversion for the `JsonAdminEvent` sent to subscribers when the `Mailbox` is used to store
-// admin events.
-#[cfg(not(feature = "admin-service-event-store"))]
-impl From<(time::SystemTime, AdminServiceEvent)> for JsonAdminEvent {
-    fn from(raw_evt: (time::SystemTime, AdminServiceEvent)) -> Self {
-        Self {
-            timestamp: raw_evt.0,
-            event: raw_evt.1,
-            event_id: None,
-        }
-    }
-}
-
 // Conversion for the `JsonAdminEvent` sent to subscribers when the `AdminServiceStore` is
 // used to store admin events.
-// `timestamp` is set to the `UNIX_EPOCH` value to allow for backward-compatibility, as the
+// `timestamp` is set to the current time to allow for backward-compatibility, as the
```

Comment:
Is this change intentional?

Reply:
Yes, because that comment was not valid before as the timestamp was being set to the current time (using `SystemTime::now()`) rather than the `UNIX_EPOCH` time value.